### ISSUE TITLE
SCM/Explorer: Cannot get contextmenu popup with keyboard (Shift-F10)

### ISSUE THREAD

**Issue by cleidigh** [Microsoft/vscode#68811]
Keyboard command: contextmenu stopped working in both SCM and Explorer trees.
The menu pops up correctly with the mouse right click.

Version: 1.32.0-insider (user setup)
Commit: 003521e715f2dbcf4f98be99a923097505859954
Date: 2019-02-15T06:16:29.903Z
Electron: 3.1.3
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17134

**Comment by isidorn** [comment]
I can repro. This is tree specific, leaving up to @joaomoreno

**Comment by cleidigh** [comment]
@joaomoreno 
Just observed, probably obvious also, problem output tree exhibits the same behavior further indicating it's probably a base tree event issue.

**Comment by joaomoreno** [comment]
Broken by https://github.com/Microsoft/vscode/commit/e50ef11ff189afa60ced0bfab7156d843f3881ba

**Comment by joaomoreno** [comment]
This was an insanely hard bug to fix. Good catch.
